packages/transformer/package.json

```diff
@@ -53,25 +53,25 @@
     "You can find a script to see the latest @itwin/imodel-transformer version for your iTwin.js version in the README"
   ],
   "peerDependencies": {
-    "@itwin/core-backend": "4.3.0-dev.31",
-    "@itwin/core-bentley": "4.3.0-dev.31",
-    "@itwin/core-common": "4.3.0-dev.31",
-    "@itwin/core-geometry": "4.3.0-dev.31",
-    "@itwin/core-quantity": "4.3.0-dev.31",
-    "@itwin/ecschema-metadata": "4.3.0-dev.31"
+    "@itwin/core-backend": "4.3.3",
```

Comment:
shouldn't these versions be `^4.3.3`?
Also double check that our lockfile doesn't contain two copies of @itwin/core-backend now

Reply:
I changed both, I only see one hash for @itwin/core-backend. I think we're good.